server/controllers/work/editWork.js

```diff
@@ -0,0 +1,39 @@
+const { Work } = require('../../database/models');
+const { customError } = require('../errors');
+const { workEditValidation } = require('../../utils/validation');
+const { uploadImage, deleteImage } = require('../../utils');
+
+const editWork = async (req, res, next) => {
+  try {
+    const { id } = await workEditValidation.validateAsync(req.body);
+    const { title, content } = req.body;
+    const { providerID } = req;
+    const work = await Work.findByPk(id);
+    if (!work) {
+      throw customError('work does not exist', 400);
+    }
+    if (work.userId !== +providerID) {
+      throw customError('Unauthorized', 401);
+    }
+    let imgUrl = '';
+    if (req.files) {
+      const { image } = req.files;
+      imgUrl = await uploadImage(image, work.userId);
+      deleteImage(work.image);
+    }
+
+    await work.update({
+      title,
+      content,
+      image: imgUrl.Location,
```

Comment:
if not req.files, then imgUrl will be an empty string, and `imgUrl.Location` will throw an error,
And you need to set the image value if it's not send with req.files from the request body